src/tbp/monty/frameworks/config_utils/make_dataset_configs.py

```diff
@@ -1102,3 +1104,279 @@ class FiveLMMountConfig:
     zooms: List[float] = field(
         default_factory=lambda: [10.0, 10.0, 10.0, 10.0, 10.0, 1.0]
     )
+
+
+"""
+Utilities for generating multi-LM dataset args.
+"""
+
+
+def make_sensor_positions_on_grid(
+    n_sensors: int,
+    delta: Number = 0.01,
+    order_by: str = "distance",
+    add_view_finder: bool = True,
+) -> np.ndarray:
+    """Generate sensor positions on a 2D grid.
+
+    Create mounting positions for an arbitrary number of sensors, where the
+    sensors lie on an imaginary grid on the xy plane (and z = 0). Sensor position
+    0 is always centered at (0, 0, 0), and all other sensors are clustered
+    around it. The method for selecting which grid points around the center to assign
+    each sensor is determined by the `order_by` argument (see below).
+
+    By default, `n_sensors + 1` positions are returned; the first `n_sensors` positions
+    are for regular sensors, and an additional position is appended by default to
+    accommodate a view finder. The view finder position (if used) is the same as
+    sensor position 0 (i.e., (0, 0, 0)).
+
+    Args:
+        n_sensors (int): Number of sensors. Count should not include a view finder.
+        delta (number): The grid spacing length. By default, sensors will be
+            placed every centimeter (units are in meters).
+        order_by (str, optional): How to select points on the grid that will contain
+            sensors.
+             - "spiral": sensors are numbered along a counter-clockwise spiral
+                spreading outwards from the center.
+             - "distance": sensors are ordered by their distance from the center.
+                This can result in a more jagged pattern along the edges but
+                results in sensors generally more packed towards the center.
+                Positions that are equidistant from the center are ordered
+                counterclockwise starting at 3 o'clock.
+        add_view_finder (bool, optional): Whether to include an extra position module
+            at the origin to serve as a view finder. Defaults to `True`.
+
+    Returns:
+        np.ndarray: A 2D array of sensor positions where each row has positions
```

Comment:
Maybe instead of "row has" say "row encodes"? The former to me makes it sound like there's lots of information in each row, i.e. the xyz coordinates as well as other information.

Reply:
How about "each row is..."? "Encodes" sounds to me as if the (x, y, z) positions are encoded in an unexpected way (like a bunch of bits or something).